src/js/page_sections/Navmenu.js

```diff
@@ -19,104 +20,109 @@ class Navmenu extends Component {
   }
 
   focusOnClose = () => {
-    this.refs.closeTrigger.focus()
+    this.refs.closeTrigger.focus();
   }
 
   componentDidMount () {
-    const endpoint = NAVMENU_ENDPOINT
 
     axios
-      .get(endpoint)
+      .post(`${process.env.REACT_APP_CMS_ENDPOINT}/graphql/`, {
+        query: allTopicPagesQuery
+      })
       .then(res => {
-        const leData = res.data.items;
-        const parentNavItems = uniq(leData.map((item) => item.topic.text))
-
-        const getChildrenNavItemsByParent = (title) => {
-          return leData.filter((data) => {
-            return data.topic.text === title
-          })
-        }
-
-        const nav = parentNavItems.map((title) => {
-        	return {
-        		parentTitle: title,
-        		children: getChildrenNavItemsByParent(title),
-        	}
-        })
-
-        this.setState({ nav: nav })
+        this.setState({
+          data: res.data.data.allTopics,
+          isLoaded: true
+        });
       })
-      .catch(err => console.log(err))
+      .catch(err => console.log(err));
   }
 
   componentDidUpdate() {
     if (this.props.isOpen) {
-      this.focusOnClose()
+      this.focusOnClose();
     }
   }
 
   getCurrentPath = () => {
     // this is dependent on react-router. We may want to use
     // `window.location.href` instead.
-    return this.props.location.pathname
-  }
-
-  getMenuItemClassName = (path) => {
-    const pathname = this.getCurrentPath()
-    return pathname === path ? 'usa-current' : ''
+    return this.props.location.pathname;
   }
 
   getOverlayClassName = () => {
     let className = `coa-Navmenu__overlay`;
     if (this.props.isOpen) {
-      className = `${className} ${className}--open`
+      className = `${className} ${className}--open`;
     }
-    return className
+    return className;
   }
 
-  getParentMenuItemClassName = (topic) => {
-    const pathname = this.getCurrentPath()
+  getParentMenuItemClassName = (parentLink) => {
+    const currentPath = this.getCurrentPath();
+    const servicePaths = parentLink.services.edges
+      ? parentLink.services.edges.map(({ node:serviceLink }) => '/service/' + serviceLink.slug)
+      : [];
 
-    const currentId = Number(pathname.split('/')[2]) // this is brittle
-    const childrenIds = topic.children.map((child) => child.id)
-    const isActive = includes(childrenIds, currentId)
+    const isActive = includes(servicePaths, currentPath);
 
-    return isActive ? 'usa-current' : ''
+    return isActive ? 'usa-current' : '';
+  }
+
+  getMenuItemClassName = (path) => {
+    const currentPath = this.getCurrentPath();
+    return currentPath === path ? 'usa-current' : '';
   }
 
   render() {
-    return (
+
+    if (!this.state.isLoaded) return 'loading...';
```

Comment:
We might want to ask Celine is they have a preferred loading state. But then again, this won't matter after we go SSR or SSG so this implementation is 🆗